projects/plugins/jetpack/_inc/client/components/settings-card/index.jsx

```diff
@@ -476,12 +477,12 @@ export default connect(
 			vaultPressData: getVaultPressData( state ),
 			getModuleOverride: module_name => getModuleOverride( state, module_name ),
 			getModule: module_name => getModule( state, module_name ),
-			adsUpgradeUrl: getUpgradeUrl( state, 'settings-ads' ),
+			adsUpgradeUrl: getRedirectUrl( 'jetpack-creator-landing' ),
```

Comment:
Per this conversation: p1698846101781199-slack-C02LK1W8T4Z, I think it's preferred that these "Upgrade" CTA's end up in checkout with Jetpack Creator in the cart, instead of the Creator LP (on Jetpack.com).